rhel8/profiles/stig.profile

```diff
@@ -44,3 +44,6 @@ selections:
     - package_rsyslog-gnutls_installed
     - rsyslog_remote_tls
     - rsyslog_remote_tls_cacert
+    - sshd_rekey_limit
+    - var_rekey_limit_size=512M
+    - var_rekey_limit_time=1hour
```

Comment:
I am fine with keeping the OSPP variable changes in the STIG profile because it is such an incredibly minor change and not a drastic change.